src/Build/Microsoft.Build.csproj

```diff
@@ -26,6 +26,46 @@
     <NoWarn>$(NoWarn);NU5104</NoWarn>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net472|AnyCPU'">
```

Comment:
What's this part for? Is this different than saying `<PropertyGroup Condition="'$(TargetFramework)'=='net472'">`? I'm not sure if the mono ones are still necessary; I don't think mono msbuild has been updated at all recently...

Reply:
Yeah, mono msbuild hasn't been updated for 17.0 it won't be. See https://github.com/dotnet/msbuild/issues/7034 for details there.

Reply:
Sorry, I don't think the changes to this file should have been pushed through. I will revert them.